main/lsp/ShowOperation.h

```diff
@@ -0,0 +1,29 @@
+#ifndef RUBY_TYPER_LSP_SHOWOPERATION_H
+#define RUBY_TYPER_LSP_SHOWOPERATION_H
+
+#include <string>
+#include <string_view>
+
+namespace sorbet::realmain::lsp {
+class LSPOutput;
+class LSPConfiguration;
+
+/**
+ * Object that uses the RAII pattern to notify the client when a *slow* operation
+ * starts and ends. Is used to provide user feedback in the status line of VS Code.
+ */
+class ShowOperation final {
+private:
```

Comment:
nitpick: private is redundant here as members of classes are private by default.
Happy to keep if you'd like to make it explicit.